po/ru.po

```diff
@@ -542,6 +542,9 @@ msgstr " Kad: Выключено"
 #: src/DownloadListCtrl.cpp:652 src/FriendListCtrl.cpp:174
 #: src/muuli_wdr.cpp:802 src/muuli_wdr.cpp:862 src/muuli_wdr.cpp:936
 #: src/muuli_wdr.cpp:1002 src/muuli_wdr.cpp:2332 src/muuli_wdr.cpp:2434
+#: src/muuli_wdr.cpp:1494
+msgid "Show notifications when finished downloading"
+msgstr "Показывать уведомления о завершении загрузки"
```

Comment:
the build fails:

```
ru.po:3357: duplicate message definition...
ru.po:547: ...this is the location of the first definition
```

The translation must be done into another line which already has `"Show notifications when finished downloading"`